changing docs for config-gen (#8083)
chef/automate: components/docs-chef-io/content/automate/ha_aws_deploy_steps.md

    - If you have your Automate, ChefServer, OpenSearch, and Postgresql certificates, select `yes` and provide relevant certificates. Click [here](/automate/ha_cert_deployment) to learn more about adding service certificates during deployment.

    ```bash
        Will you use custom certs for any service like Automate, Chef Infra Server, PostgreSQL, or OpenSearch:
        > no
          yes
    ```

    - Provide `AWS profile name`. Skip this if the IAM role is configured on the bastion host.
    - Filter and select the AWS region from the list.
    - Give AWS VPC ID created in the Prerequisites step. Example: `"vpc12318h"`
    - To create subnets, we have two options: CIDR Block and subnets ids. Select `yes` for CIDR and `no` for subnet ids, and provide subnet ids created in the Prerequisites step. For example, `subnet-07e469d218301533`, subnets should be created under the same VPC provided above. We need three private subnets; if you want to keep the load balancer on public IP, we need three public subnets as well; it is recommended to use subnet ids.

    ```bash
        Do you want to use AWS CIDR Block:
        > yes
          no
    ```

>     - Give the `ssh key pair name` name used for creating the ssh key pair, `AMI Id` which depends on the AWS Region and the Operating System image you want to use.,

Reviewer: ```suggestion
    - Give the `ssh key pair name` name used for creating the ssh key pair, `AMI Id` which depends on the AWS Region and the Operating System image you want to use.
```

    - If you want to terminate all the resources on deletion, select  `on`.

    ```bash
        Delete on termination should be:
        > off
          on
    ```

    - If you want to enable access log on AWS load balancers, select `yes`.

    ```bash
        Do you want to Enable Access Logs on AWS Load Balancer:
        > yes
          no
    ```

    - Give Automate FQDN example `chefautomate.example.com`
    - Give Automate loadbalancer ARN, with the arn value of the Certificate created in AWS ACM for DNS entry of `chefautomate.example.com`.

>     - Give the path of Automate loadbalance FQDN ssl root ca-certificates

Reviewer: ```suggestion
    - Give the path of Automate loadbalancer FQDN ssl root ca-certificates
```

    - Set Chef Server instance type, recommended is `m5.large`.
    - Set Chef Server EBS volume size based on your load needs.
    - Set Chef Server EBS volume type; default is `gp3`; change as per your requirement.
    - Set Chef Server EBS volume IOPS based on your load needs.
    - Select `no` for chef-managed database deployment:

    ```bash
        Do you want to use AWS Managed Databases:
         yes
        > no
    ```

    - Set the number of Postgresql nodes you want; the recommended is three nodes
    - Set postgresql instance type, recommended is `m5.large`.
    - Set PostgreSQL EBS volume size based on your load needs.
    - Set PostgreSQL EBS volume type; default is `gp3`; change as required.
    - Set PostgreSQL EBS volume IOPS based on your load needs.
    - Set the number of OpenSearch nodes you want; the recommended is three nodes
    - Set OpenSearch instance type, recommended is `m5.large`.
    - Set OpenSearch EBS volume size based on your load needs.
    - Set OpenSearch EBS volume type; default is `gp3`; change as required.
    - Set OpenSearch EBS volume IOPs, based on your load needs.

>     - If you want to configure the database during deployment, select `yes` and provide details accordingly for the selected backup type; for S3 backup, provide details like `bucket name`.

Reviewer: ```suggestion
    - If you want to configure the backup during deployment, select `yes` and provide details accordingly for the selected backup type; for S3 backup, provide details like `bucket name`.
```

    #Download latest Airgapped Bundle.
    #To download specific version bundle, example version: 4.2.59 then replace latest.aib with 4.2.59.aib
    curl https://packages.chef.io/airgap_bundle/current/automate/latest.aib -o automate.aib
    "
    ```

    {{< note >}} Chef Automate bundles are available for 365 days from the release of a version. However, the milestone release bundles are available for download forever. {{< /note >}}

1. Generate config with relevant data using below command 
    ```bash
        chef-automate config gen config.toml
    ```
    - Select `Chef Automate HA` as topology
    ```bash
    Choose Topology:
        Chef Automate HA
    ```

>     - select deployment type as `AWS`

Reviewer: ```suggestion
    - Select deployment type as `AWS`
```

    ```bash
    Choose type of Deployment Type:
       On-Premise
     > AWS
    ```

>     - provide  `ssh user name`, `ssh group name`, `ssh port no`, `ssh key file path`

Reviewer: ```suggestion
    - Provide  `ssh user name`, `ssh group name`, `ssh port no`, `ssh key file path`
```

>     - if you have own certificates for Automate, ChefServer, OpenSearch and Postgresql, then select `yes` and provide relevant certificates. Click [here](/automate/ha_cert_deployment) to know more on adding certificates for services during deployment.

Reviewer: ```suggestion
    - If you have own certificates for Automate, ChefServer, OpenSearch and Postgresql, then select `yes` and provide relevant certificates. Click [here](/automate/ha_cert_deployment) to know more on adding certificates for services during deployment.
```

Reviewer: Follow the case convention for all below steps

    ```bash
        Will you use custom certs for any service like Automate, Chef Infra Server, PostgreSQL, OpenSearch:
        > no
          yes
    ```

>     - provide `AWS profile name` skip this if IAM role configured on bashtion host

Reviewer: ```suggestion
    - provide `AWS profile name` skip this if IAM role configured on Bastion host
```

    - filter and select AWS region from list
    - give AWS VPC ID created in the Prerequisites step. Example: `"vpc12318h"`

>     - to create subnets we have two options as CIDR Block and subnets ids, select  `yes` for CIDR and `no` for subnet ids, and provide subnet ids created in the Prerequisites step. Example `subnet-07e469d218301533`, subnets should be created under same VPC provided above, we need three private subnets, if you want to keep loadbalancer on public IP then we need three public subnets as well, recomended is to use subnet ids.

Reviewer: This point is suggesting to choose yes for `CIDR` but then its suggesting to provide subnet ids which is wrong,If this point is about using subnets instead of CIDR then it should be mentioned like `No` for CIDR and `Yes` for subnet ids

>     - to create subnets we have two options as CIDR Block and subnets ids, select  `yes` for CIDR and `no` for subnet ids, and provide subnet ids created in the Prerequisites step. Example `subnet-07e469d218301533`, subnets should be created under same VPC provided above, we need three private subnets, if you want to keep loadbalancer on public IP then we need three public subnets as well, recomended is to use subnet ids.

Reviewer: ```suggestion
    - to create subnets we have two options as CIDR Block and subnets ids, select  `yes` for CIDR and `no` for subnet ids, and provide subnet ids created in the Prerequisites step. Example `subnet-07e469d218301533`, subnets should be created under same VPC provided above, we need three private subnets, if you want to keep loadbalancer on public IP then we need three public subnets as well, we recommend to use subnet ids.
```
